stac_fastapi/pgstac/stac_fastapi/pgstac/app.py

```diff
@@ -27,6 +28,7 @@
         QueryExtension(),
         SortExtension(),
         FieldsExtension(),
+        ContextExtension(),
```

Comment:
pgstac does not currently support the context extension, and so shouldn't include it by default in the current implementation. However, the newest releases of pgstac do support this extension. Once we bump up the pgstac version we could potentially include this.

Reply:
Thanks @lossyrob

The new version seems not to support context extension for pgstac. I have looked at implementation on sqlalchemy core.py and seems I can dirt hands on this. Let me know if you need an extra hand!